beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -101,22 +103,29 @@ public Consumer<BeaconBlockBodyBuilder> createSelector(
       final SszList<AttesterSlashing> attesterSlashings =
           attesterSlashingPool.getItemsForBlock(
               blockSlotState,
-              slashing -> !exitedValidators.containsAll(slashing.getIntersectingValidatorIndices()),
+              slashing ->
+                  !exitedValidators.containsAll(slashing.getIntersectingValidatorIndices())
+                      && spec.computeEpochAtSlot(slashing.getAttestation1().getData().getSlot())
+                          .isGreaterThanOrEqualTo(minimalViableEpoch),
```

Comment:
nit: Could probably simplify this by calculating the start slot of minimalViableEpoch ahead of time and just comparing to that rather than having to convert slot to epoch here.